src/engine/renderer/tr_image.cpp

```diff
@@ -2692,8 +2692,13 @@ static void R_CreateShadowCubeFBOImage()
 // *INDENT-ON*
 
 // *INDENT-OFF*
-static void R_CreateBlackCubeImage()
+static void R_CreateReflectionCubeImages()
 {
+	if ( !r_reflectionMapping->integer || !glConfig2.specularMapping )
```

Comment:
None of the other code checks specularMapping when deciding whether to use reflection mapping.

Reply:
This commit was removed, and this function now only generates blackCube which is now expected to be used by the sky code anyway.